docs/swarm_at_scale/about.md

```diff
@@ -56,7 +56,7 @@ Behind the frontend is a worker tier which runs on separate nodes. This tier:
 Just like the frontend, the worker tier can also scale arbitrarily. The worker
 count and frontend count are independent from each other.
 
-The applications dockerized microservices are deployed to a container network.
+The application's dockerized microservices are deployed to a container network.
```

Comment:
@ogryzek while you're at it, could you also change `dockerized` -> `Dockerized`?